src/Jackett.Common/Definitions/hdbao.yml

```
@@ -0,0 +1,297 @@
+    # 0: include dead, 1: alive, 2: dead
+    incldead: 0
+    spstate: "{{ if .Config.freeleech }}2{{ else }}0{{ end }}"
+    # 0: title, 1: description, 3: uploader, 4: IMDB URL
+    search_area: "0"
+    # 0: AND, 2: exact
+    search_mode: 0
+    sort: "{{ .Config.sort }}"
+    type: "{{ .Config.type }}"
+    # TODO: The example I copied, `./freefarm.yml`, includes a
+    # `notnewword` key but I don't see that key from submitting the
+    # search form in the web UI of either that example or this
+    # indexer. What am I missing?
+    #
+    # TODO: On that note, while inspecting the payloads of both search
+    # forms I noted a bunch of parameters that the web UI sends that
+    # aren't included in that example. How does that reconcile with
+    # "we don't want an obvious 'Jackett footprint'" from my last PR
+    # feedback especially given how much of Jackett/Prowlarr's traffic
+    # is submissions of this form?
+
+  rows:
+    selector: table.torrents > tbody > tr:has(a[href^="download.php?id="])
+
+  fields:
+    category:
+      selector: a[href^="?cat="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: cat
+    title_default:
+      selector: a[href^="details.php?id="]
+    title:
+      selector: a[title][href^="details.php?id="]
+      attribute: title
+      optional: true
+      default: "{{ .Result.title_default }}"
+    details:
+      selector: a[href^="details.php?id="]
+      attribute: href
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    poster:
+      selector: img[data-src]
+      attribute: data-src
+    imdbid:
+      # site currently only has a badge and rating, the id is not present. just in case a future update.
+      selector: a[href*="imdb.com/title/tt"]
+      attribute: href
+    doubanid:
+      # site currently only has a badge and rating, the id is not present. just in case a future update.
+      selector: a[href*="movie.douban.com/subject/"]
+      attribute: href
+    date_elapsed:
+      # TODO: The selector from the example I copied includes
+      # `.rowfollow` which is a class for most of the column cells and
+      # AFAICT is only for presentation. As such I think it makes the
+      # selector more fragile and I've removed it from this
+      # definition. Should it be cleaned up in other definitions?
+      #
+      # time type: time elapsed (default)
+      selector: td:nth-child(4) > span[title]
+      attribute: title
+      optional: true
+      filters:
+        - name: append
+          # TODO: This is another demonstration of why a more verbose comment is
+          # valuable here. The `CST` abbreviation is used for both `-04:00` through
+          # `-06:00` in the Americas *and* `+08:00` in Asia. The more verbose comment
+          # removes ambiguity and saves contributor time.
+          args: " +08:00" # Asia/Taipei, CST
```

```suggestion
          args: " +08:00" # CST
```

The combination of the time difference and the time zone abbreviation is enough to make that distinction. This conversation has been had, thanks.